### ISSUE TITLE
Reproduced Results

### ISSUE THREAD

**Issue by Bohemianc** [kswoo97/GLAD_MUSE#1]
Hi, thanks for releasing the code for this work.

I ran your code using the command provided `python3 main.py -data dd -anom_type 0 -lr 0.001 -dim 256 -n_layers 4 -gamma 2.0 -device cuda:0 ` and obtained the following results:
```
LR: 0.001 | Hid-Dim: 256 | Layers: 4 | Gamma: 1.0
Data: dd
Average AUROC: 0.5344 ± 0.0711
Average AP: 0.6608 ± 0.0365
Average TopK: 0.40 ± 0.14
```

However, the paper reports an AUROC for DD dataset of **0.805** for this setting, which is significantly higher.

Could you please clarify whether there are any additional settings, preprocessing steps, or experimental details needed to reproduce the reported result?

**Comment by kswoo97** [comment]
Thank you for your question. I suspect the hyperparameters are not being applied correctly.

1. In the code, gamma is set to 2.0, but the printed output indicates that gamma = 1.0 is being used. Please verify that the hyperparameters passed via argparse are actually propagated into the model configuration (and not being overwritten by defaults).

2. Please also confirm that the hyperparameters stored in MUSE_hyperparameters.pickle are being loaded and used during training/inference. It’s possible the pickle file is overriding the values you expect, or not being read at all.

**Comment by Bohemianc** [comment]
Thank you for the clarification. You were right — the issue was indeed caused by the hyperparameters not being applied correctly. After explicitly setting `gamma = 2.0`, the results now match those reported in the paper.

I also really appreciate the idea in your work, especially the observation of the reconstruction flip phenomenon in GLAD. It was really insightful. Thank you for your contribution.
